core/src/main/scala/com/itv/bucky/wiring/Wiring.scala

```diff
@@ -56,7 +56,17 @@ class Wiring[T](
   def publisherDeclarations: List[Declaration] =
     List(exchange)
   def consumerDeclarations: List[Declaration] =
-    List(exchangeWithBinding) ++ requeue.requeueDeclarations(queueName, dlxRoutingKey, Some(ExchangeName(s"${queueName.value}.dlx")), dlxType)
+    List(exchangeWithBinding) ++ requeue.requeueDeclarations(queueName,
+                                                             dlxRoutingKey,
+                                                             Some(ExchangeName(s"${queueName.value}.dlx")),
+                                                             dlxType,
+                                                             requeueRetryAfter)
+
+  //retain the default requeue ttl of 5 minutes, unless requeue policy requires a longer delay
+  private def requeueRetryAfter: FiniteDuration =
```

Comment:
Retain requeue ttl of 5 mins if requeue policy is 5 minutes or less as rabbit will choose the lower of the two values (current behaviour). Otherwise set requeue ttl to match the requeue policy.